tools/ci/README.md

```diff
@@ -0,0 +1,11 @@
+Testing Docker Image
+====================
+
+Local testing of the [Docker image](Dockerfile) used in [Jenkins](Jenkinsfile) can be performed by
```

Comment:
Just "Jenkins" is confusing since we are referring to a file in this repo and not a Jenkins server.
Suggested fix:

`used in Jenkins` -> `used by the Jenkinsfile`

Reply:
Changed to suggested fix.